generators/app/templates/gulp/_building.js

```diff
@@ -21,11 +25,28 @@ gulp.task('build', buildDependencies, function () {
     .pipe($.size({showFiles: true}));
 });
 
-gulp.task('clean', function () {
-  return gulp.src(['.tmp', paths.dist + '/*'])
+gulp.task('clean', ['clean-styles','clean-templates'], function () {
```

Comment:
I wish I could do it, but then jade support won't work well. After any change of jade file you will clean up sass files and vice versa. This needs to be split. Or do you have any better solution?